src/helper.cc

```diff
@@ -430,7 +431,8 @@ statefulhelper::openSessions()
         if (wfd != rfd)
             commSetNonBlocking(wfd);
 
-        AsyncCall::Pointer closeCall = asyncCall(5,4, "helper_stateful_server::HelperServerClosed", cbdataDialer(helper_stateful_server::HelperServerClosed, srv));
+        AsyncCall::Pointer closeCall = asyncCall(5, 4, "helper_stateful_server::HelperServerClosed", cbdataDialer(Helper::SessionBase::HelperServerClosed,
```

Comment:
If possible, please use callbackDialer() here instead of cbdataDialer(), converting Helper::SessionBase::HelperServerClosed() into a non-static class method.

Same for the other helper class.